docs/reference/end-to-end-planning.md

```diff
@@ -103,7 +103,10 @@ Parameters:
 
```

Comment:
There's a link to the learner on line 12 that's broken in a similar fashion as the others that i missed earlier, need to add `../../`.

Reply:
added